cpp/include/cudf/strings/detail/utilities.cuh

```diff
@@ -25,13 +25,13 @@
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/exec_policy.hpp>
 
+#include <dlfcn.h>
+#include <mutex>
```

Comment:
Can  you move these includes down next to `#include <unordered_map>` ?

Reply:
`clang-format` seems to want to pull them back up here?

Reply:
The empty line is important.
```
#include <thrust/distance.h>
#include <thrust/for_each.h>
#include <thrust/iterator/counting_iterator.h>
#include <thrust/iterator/transform_iterator.h>
#include <thrust/scan.h>

#include <dlfcn.h>
#include <mutex>
#include <unordered_map>
```

Reply:
Thanks! Fix'd;